03-first-aws-environment/stacks/catalog/globals.yaml

```diff
@@ -5,8 +5,8 @@ terraform:
   backend:
     s3:
       encrypt: true
-      bucket: "acme-ue2-tfstate-TODO"
+      bucket: "acme-ue2-root-tfstate-TODO"
       key: "terraform.tfstate"
-      dynamodb_table: "acme-ue2-tfstate-lock-TODO"
+      dynamodb_table: "acme-ue2-root-tfstate-lock-TODO"
```

Comment:
I think this is a single account tutorial, no?

Reply:
this example has root, dev, and prod